src/backend/common/queries/dict_converters/event_details_converter.py

```diff
@@ -27,15 +25,6 @@ def eventsDetailsConverter_v3(cls, event_details: List[EventDetails]):
 
     @classmethod
     def eventDetailsConverter_v3(cls, event_details: EventDetails) -> EventDetailsDict:
-        normalized_oprs = defaultdict(dict)
-        if event_details and event_details.matchstats:
-            for stat_type, stats in event_details.matchstats.items():
-                if stat_type in {"oprs", "dprs", "ccwms"}:
-                    for team, value in cast(Dict[TeamKey, float], stats).items():
-                        if "frc" not in team:  # Normalize output
-                            team = "frc{}".format(team)
```

Comment:
Are we worried about removing this normalization? I'm assuming there's a reason this code exists. Especially with the `keyed` optional method parameter.

Reply:
I've changed match stats to always return keyed values. I believe that should be sufficient, but I can add logic back into here if you would like.